ehsm_kms_service/router.js

```diff
@@ -56,7 +56,7 @@ const router = async (p) => {
   const action = req.query.Action
   switch (action) {
     case enroll_apis.RA_GET_API_KEY:
-      create_user_info(DB, res)
+      create_user_info(action, DB, res, req)

```

Comment:
why add action param? the action only can be RA_GET_API_KEY, right?

Reply:
old function is a sample, the action on line 59 is RA_GET_API_KEY.

Reply:
if the action only can be RA_GET_API_KEY, why need to add a parameter for it?

Reply:
In line 57, the switch action determines that the action contains RA_GET_API_KEY,create_user_info needs to be used, so it is directly passed in to reduce handcode.